InformationScripting/src/queries/AstQuery.cpp

```diff
@@ -285,6 +297,42 @@ TupleSet AstQuery::usesQuery(TupleSet input)
 	return result;
 }
 
+TupleSet AstQuery::typeFilter(TupleSet input)
+{
+	using SymbolType = Model::Node::SymbolType;
+
+	SymbolType expectedSymbolType = SymbolType::VARIABLE;
+	// FIXME don't use nodetype argument as it is the actual type:
+	QString expectedType;
+	QStringList arguments;
+	auto typeArgument = argument(NODETYPE_ARGUMENT_NAMES[1]);
+	// NOTE: no spaces allowed for this argument.
```

Comment:
I'm not sure which argument, and why we have this restriction.

In general, could you write a few examples (in a comment in the PR) of how to use this filter? What's allowed, what's not allowed?